admission-management.yaml

```diff
@@ -105,7 +105,7 @@ paths:
             application/json: 
               schema:
                 $ref: '#/components/schemas/GenericError'
-  /admissions/courses/proposals/add:
+  /admissions/courses/add_proposal:
```

Comment:
should we prefix this with unsigned to be consistent with the matriculation api?

Reply:
Good Point, is now changed accordingly